src/main/java/uk/gov/companieshouse/config/MongoConfig.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.companieshouse.config;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@RequiredArgsConstructor
+public class MongoConfig {
+
```

Comment:
What do you plan this config for? Is that needed? How will mongo DB configured?

Reply:
I don't think this is needed right now as, it's only if we wanted more control over the connection to the database. I'll remove this for now and then if we find we want to override any of the out of box configurations then we can do this later